docs/source/tutorials/kagome_in_docker.md

```diff
@@ -0,0 +1,66 @@
+## Runing Kagome in docker container
+
+In this tutorial you will learn how to execute Kagome-based Polkadot-host chain which can be used as a cryptocurrency, and interact with it by sending extrinsics and executing queries.
+
+### Prerequisites
+
+1. Docker container runtime must be installed in your OS
+   ```bash
+   # In Debian-like OS (Debian, Ubuntu, Mint, etc.)
+   apt-get install docker.io
```

Comment:
As far as I understand this is obsolete now.
https://docs.docker.com/engine/install/ubuntu/#uninstall-old-versions

Reply:
Agree. Replaced by lint to docker's official site